### DISCUSSION TITLE
v7.0.0

### DISCUSSION THREAD

**Discussion by tadashi-aikawa** [tadashi-aikawa/obsidian-another-quick-switcher#101] [Announcements]
Done.

https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/7.0.0

**Comment by FelipeRearden** [comment]
Hello @tadashi-aikawa !!!!

Thanks for a new beta release !!!!!

**Question 🧐** I just wanna ask you about something ...

I am not an insider, this way I dont have access to the latest beta public `0.16.2` 😭

This way, I wanna ask you if would be valuable for testing purposes if I **manually change manifest.json to 0.15.9 and test v7.0.0-beta** and do the tests on Desktop.

Let me know if I can proceed this way :)

**Reply by tadashi-aikawa** [reply]
Hi, @FelipeRearden :)

Another Quick Switcher 7.0.0 is not compatible with Obsidian 0.15. So I think you should not test the beta version.

Please wait for Obsidian's v0.16.0, or get the insider one by being a Catalyst license :trollface:

**Reply by FelipeRearden** [reply]
> Another Quick Switcher 7.0.0 is not compatible with Obsidian 0.15. So I think you should not test the beta version.

100% understood ! 

Forgive me for not being able to do the tests on beta 🙏🙏🙏 😭😭😭

It's the first time that I cant help with the tests but I promise that I will do the tests on Desktop and Mobile the second after the new version is released 😇   

> Please wait for Obsidian's v0.16.0, 

Okay ! I have to control my anxiety until the new release 😬😬😬  

> or get the insider one by being a Catalyst license :trollface: 

😜😜😜 

---

Thanks for everything @tadashi-aikawa !!!

**Reply by FelipeRearden** [reply]
Hello @tadashi-aikawa !!!!! 

Just to let you know that I will test these new features on mobile in the next few weeks 👀

I always wait a few weeks before updating to big versions like `0.16 → 1.0` because I have to make sure that all my plugins and css snippets are going to work 🙏
- On mobile we cant have different vaults with different versions. It's always **all in** :)

I am writing to you just to let you know that I am here and will test the new features of `v7.x`

Have a great day!!!

**Reply by FelipeRearden** [reply]
> I always wait a few weeks before updating to big versions like `0.16 → 1.0` because I have to make sure that all my plugins and css snippets are going to work 🙏

Finally updated to v1.0 on Mobile. I am gonna do the tests until the end of the weekend :)

**Comment by tadashi-aikawa** [comment]
I have released [v7.0.0-beta8](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/7.0.0-beta8) which seems to be the last beta version if there are no bugs.

To be honest, I'm not sure how it would work on macOS correctly😅. If anyone uses macOS and BRAT, it would be great to try the beta version 🙏

**Comment by tadashi-aikawa** [comment]
Since it seems more times until releasing Obsidian v0.16.0, I have released [v7.0.0-beta9](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/7.0.0-beta9), which includes some new features 🚀 Especially, `Integrate "Backlink search" into custom searches` .
